README.md

```diff
@@ -34,7 +34,15 @@ The `install-dev.sh` script will build and push an image of the Tekton Dashboard
 3. Install deployment config `$oc process -f config/templates/deploy.yaml | oc apply -f-`
 4. Install build config `$oc process -f config/templates/build.yaml | oc apply -f-`
 
+## Install on Minishift
+
+1. Install [tektoncd-pipeline-operator](https://github.com/openshift/tektoncd-pipeline-operator#deploy-openshift-pipelines-operator-on-minikube-for-testing)
+2. [Checkout](https://github.com/tektoncd/dashboard/blob/master/DEVELOPMENT.md#checkout-your-fork) the repository
+3. Install deployment config `$ oc process -f config/templates/deploy.yaml | oc apply -f-`
+4. Install build config `$ oc process -f build.yaml | oc apply -f-`
```

Comment:
Is this meant to point to `config/templates/build.yml` or preferably `config/templates/build.yaml` the extensions for `deploy.yaml` and `build.yml` should probably be consistent

Reply:
This is meant to point to `build.yaml` to stay consistent.